CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+## 2.0.4
+ - Elasticsearch 2.0 does not allow for dots in field names.  This change changes to use sub-field syntax instead of
```

Comment:
Seems like the version number should change more wildly with such a breaking change. Thoughts?